gateway/api/src/main/java/eu/openg/guessnumberapi/gateway/api/GameRepository.java

```diff
@@ -0,0 +1,13 @@
+package eu.openg.guessnumberapi.gateway.api;
+
+import eu.openg.guessnumberapi.domain.Game;
+
+public interface GameRepository {
+    int saveNewGameAndReturnId(Game game);
+
+    int incrementAndReturnGuessCount(int gameId);
+
+    Game fetchGame(int gameId);
+
+    void closeConnection();
```

Comment:
SOL**I**D

Reply:
šitas interface neturėtų turėt metodo susijusiu su duomenų bazės connection valdymu. Nes matei, dabar pas tave inMemoryRepo turi nelogišką metodą closeConnection

Reply:
gali išsikelt šitą connection valdymą į atskirą klasę